StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/v1-elements-sessions/PaymentMethodWithLinkDetails.swift

```diff
@@ -29,9 +29,16 @@ class PaymentMethodWithLinkDetails: NSObject, STPAPIResponseDecodable {
             return nil
         }
 
+        let linkDetailsJson = response["link_payment_details"] as? [AnyHashable: Any]
```

Comment:
Question: Why not do this check after attempting deserialization?

Reply:
Oh right, I totally can. Have been following my [Android solution](https://github.com/stripe/stripe-android/pull/10799) too closely, where things are a little different.